frontend/src/features/leases/detail/LeasePages/surplus/Surplus.tsx

```diff
@@ -0,0 +1,86 @@
+import { FormSection } from 'components/common/form/styles';
+import { ColumnWithProps, Table } from 'components/Table';
+import { getIn, useFormikContext } from 'formik';
+import { ILease, IProperty } from 'interfaces';
+import { prettyFormatDate } from 'utils';
+import { withNameSpace } from 'utils/formUtils';
+
+interface IDeclaration {
+  id: number;
+  identifier: string;
+  declarationType: string;
+  date: string;
+  comments: string;
+}
+
+const columns: ColumnWithProps<IDeclaration>[] = [
+  {
+    Header: 'PID / Identifier',
+    accessor: 'identifier',
+    maxWidth: 40,
+  },
+  {
+    Header: 'Surplus Declaration',
+    accessor: 'declarationType',
+    maxWidth: 40,
+    Cell: ({ row: { original } }) => {
+      const declarationType = original.declarationType;
+      const isYesDeclarationType = declarationType.toUpperCase() === 'YES';
+      return (
+        <>
+          {isYesDeclarationType ? (
+            <strong className="text-success">{declarationType}</strong>
+          ) : (
+            <span>{declarationType}</span>
+          )}
+        </>
+      );
+    },
+  },
+  {
+    Header: 'Declaration Date',
+    accessor: 'date',
+    Cell: ({ cell: { value } }) => prettyFormatDate(value),
+    maxWidth: 50,
+  },
+  {
+    Header: 'Declaration Comments',
+    accessor: 'comments',
+    minWidth: 150,
+  },
+];
+
+const Surplus: React.FunctionComponent<{}> = () => {
+  const { values } = useFormikContext<ILease>();
+  const organizations: IProperty[] = getIn(values, withNameSpace(undefined, 'properties')) ?? [];
```

Comment:
I think here `withNameSpace()` shouldn't take undefined but instead receive it via props like **Tenant.tsx** (line 21).
The namespace values allow to create sub-forms from a big complex Formik object. 
A quick scan of related code tell me that so far we are not nesting the forms (but instead consuming form fields from the main form) but if the other sub-components of the whole Lease page are build with namespacing, this one should follow in my opinion.

Any thoughts @devinleighsmith ?

Reply:
Yeah, chatted about that one and although the other components are using the property, they are mostly set to undefine since the parent does not set it. My opinion would be to set it to undefined if is not currently set on the parent.

Reply:
I'd rather add it to the parent or remove altogether on every sub-form. Namespacing adds complexity to the code and if it will not be used (because all fields will be on the main formik form) then better to remove `withNameSpace()` from the new forms

Reply:
```tsx
withNameSpace(undefined, 'properties') === 'properties'
```
So better to just use the string instead

Reply:
updated